IonicPortals/IonicPortals/PortalUIView.swift

```diff
@@ -109,6 +115,10 @@ public class PortalUIView: UIView {
     
 }
 
+extension Collection {
+    var isNotEmpty: Bool { !isEmpty }
```

Comment:
A real purist would be upset with this extension but I think it's lazy this isn't in the regular API. This is much better for `if ([true statement])` code style